compute_tools/src/bin/compute_ctl.rs

```diff
@@ -482,6 +484,11 @@ fn cli() -> clap::Command {
                 )
                 .value_name("FILECACHE_CONNSTR"),
         )
+        .arg(
+            Arg::new("file-cache-on-disk")
+                .long("file-cache-on-disk")
+                .action(clap::ArgAction::SetTrue),
```

Comment:
BTW, if you prefer to call this `file-cache-in-memory`, I think you could do that if you use `default_value` here to make true if omitted. Based on the TODO comment 20 lines earlier, I think that's what was done in similar situations before

Reply:
Hm, I'm not too familiar with clap's semantics around flags without/with values — honestly I think it's ok enough, because we can make on-disk the default behavior and then switch to providing `file-cache-in-memory` as console releases happen over the next few weeks.